pkg/autoscaler/multiscaler.go

```diff
@@ -289,6 +289,13 @@ func (m *MultiScaler) createScaler(ctx context.Context, decider *Decider) (*scal
 		pokeCh:  make(chan struct{}),
 	}
 	runner.decider.Status.DesiredScale = -1
+	runner.decider.Status.ExcessBurstCapacity = func() int32 {
```

Comment:
Why not only set it if `TBC == -1`? i.e.

```golang
if runner.decider.Spec.TargetBurstCapacity == -1 {
  runner.decider.Status.ExcessBurstCapacity = -1
}
```

should be semantically the same? I'd love `if` to be an expression though 😋

Reply:
We might want to move to a better logic in future.
IDK. If you want that way -- I'm find with changing.